pynest/examples/pynest-microcircuit/readme.md

```diff
@@ -0,0 +1,29 @@
+# Microcircuit Readme
+
+Authors: Hendrik Rothe, Hannah Bos, Sacha van Albada   
+May 2016
+
+## Description ##
+This is a PyNEST implementation of the SLI cortical microcircuit from the microcircuit model by Potjans and Diesmann (2014): The cell-type specific
+cortical microcircuit: relating structure and activity in a full-scale spiking
+network model. Cerebral Cortex: doi:10.1093/cercor/bhs358  
+
+* Files:
+	* network.py  
+	file which gathers all parameters and connects the different nodes with each other
+	* network\_params.py  
+	contains the parameters for the network
+	* sim\_params.py  
+	contains the simulation parameters
+	* stimulus\_params.py  
+	contains the parameters for the stimuli
+	* example.py  
+   use this script to try out the microcircuit
+   
+How to use the example:
+
+To run the microcircuit on a local machine, adjust the variables N\_scaling and K\_scaling in network_params.py to 0.1 and create an output directory called '/data'. Then run 'example.py' with python. The output will be saved in the directory '/data'.
```

Comment:
give an explanation what N_scaling and K_scaling mean and state that the full network refers to the value 1